source/admin_manual/list_of_events.rst

```diff
@@ -2333,6 +2333,38 @@ happens.
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
 
+fs_dictmap_max_bucket_changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+This event is sent whenever the ``max_bucket`` value for a mailbox changes.
+There can be two situations this happens. Either a new mail is added to a
+mailbox, where the current bucket is found to be filled and a next bucket is
+started to get filled (``reason == file``).
+
+Besides the latter, expected situation, Dovecot emits this event if it
+encounters a bucket with a higher index then the current max_bucket while
+iterating a mailbox (``reason == iter``).
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| success               | Updating ``max_bucket`` in dict & fs-bucket-cache was|
+|                       | successful: ``yes`` or ``no``.                       |
```

Comment:
Generally we don't have "success=no", just a missing success field.

Reply:
I removed the success field as the absence of "error" implies success.